setup.py

```diff
@@ -0,0 +1,20 @@
+#!/bin/env python
+from setuptools import setup,find_packages
+
+setup(
+    name="pomodorotasks",
+    packages=find_packages(exclude=['test' 'test/*']),
+    install_requires=['tasklib', 'pygobject', 'dbus-python'],
+    entry_points={
+        'console_scripts': [
+            'pomodorotasks = pomodoro.client:main'
+            ],
+        'gui_scripts': [
+            'pomodoro-taskd = pomodorotasks.daemon:main',
+            ]
```

Comment:
The /usr/bin scripts should be installed as [entry points](http://setuptools.readthedocs.io/en/latest/setuptools.html#automatic-script-creation) so I can still rename the module files if you need them to, but it should not be necessary for that. Personally I think `pomodorotasks/deamon.py` works well enough but I'll leave that to you. 

I just noticed the client is called pomodorotasks without the hyphen though, maybe I should address that.